src/scss/yellow/components/form/_steps.scss

```diff
@@ -20,9 +20,32 @@
     flex: 1;
     font-size: $size-font-xxs;
     line-height: $size-line-height-xxs;
-    padding: $size-spacing-xxs;
+    padding: 17px;
```

Comment:
Qual o motivo para não respeitarmos o token aqui?

Reply:
In this case (media: 360px), plus 1px to reach the tokien (18px) will make difference in the page look. I remember I showed you another example and we both agreed to not use the tokien to not break the page look.

Reply:
I understand, but as we are talking about the new identity all the spacing would have to follow Niten's tokens, if we don't fall into the same mistake we made back there with Gaiden, of not having a solid identity and a lot of broken value.

I suggest that to fix this communicate the Designers about this fact and if we need to have one more token or align so that it can be done in existing tokens.